src/rpc/mining.h

```diff
@@ -6,6 +6,6 @@
 #define BITCOIN_RPC_MINING_H
 
 /** Default max iterations to try in RPC generatetodescriptor, generatetoaddress, and generateblock. */
-static const uint64_t DEFAULT_MAX_TRIES{1000000};
```

Comment:
May skip this and use the isNull() approach. Alternatively, add 
```C++
uint64_t get(uint64_t default_value) { return isNull() ? (uint64_t)get_int64() : default_value; }
```
and use that.